tests/bluetooth/bsim_bt/bism_test_noisy_env/script/noisy_env.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+# EATT test
+simulation_id="noisy_env"
+process_ids=""; exit_code=0
+function Execute(){
+  if [ ! -f $1 ]; then
+    echo -e "ERR! \e[91m`pwd`/`basename $1` cannot be found (did you forget to\
+ compile it?)\e[39m"
+    exit 1
+  fi
+  # timeout 30 $@ & process_ids="$process_ids $!"
+  $@ & process_ids="$process_ids $!"
+  echo "Running $@"
+}
+: "${BSIM_OUT_PATH:?BSIM_OUT_PATH must be defined}"
+#Give a default value to BOARD if it does not have one yet:
+BOARD="${BOARD:-nrf52_bsim}"
+testing_apps_loc="${ZEPHYR_BASE}/tests/bluetooth/bsim_bt/bism_test_noisy_env"
+central_app_name="central_hr"
+peripheral_app_name="peripheral_hr"
+bsim_central_exe_name="bs_nrf52_bsim_bluetooth_central"
+bsim_peripheral_exe_name="bs_nrf52_bsim_bluetooth_peripheral"
+cd ${testing_apps_loc}
+if [ ! -d "${central_app_name}" -o ! -d "${peripheral_app_name}" ]; then
+    echo -e "ERR! \e[91mOne or more test applications couldn't be found\e[39m"
+    exit 1
+fi
+#Remove old builds if they exist
+find . -type d -name 'build' -exec rm -rf {} +
```

Comment:
1. Please use the compile.sh script to build your app (you can see there other examples)
you should not be building and (less) deleting folder or build results from the script that runs the test 
2. you need to sign off the commit, add a comment message body, and correct all the checkpatch violations

Reply:
> 1. Please use the compile.sh script to build your app (you can see there other examples)
>    you should not be building and (less) deleting folders or build results from the script that runs the test
> 2. you need to sign off the commit, add a commit message body, and correct all the checkpatch violations

1- Done; new script added for compiling `_compile.sh` and `run_noisy_env.sh` for running the tester
2- Done